lib/rubocop/cop/rails/freeze_time.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Identifies usages of `travel_to` with an argument of the current time and
+      # change them to use `freeze_time` instead.
+      #
+      # @example
+      #   # bad
+      #   travel_to(Time.now)
+      #   travel_to(Time.new)
+      #   travel_to(DateTime.now)
+      #   travel_to(Time.current)
+      #   travel_to(Time.zone.now)
+      #   travel_to(Time.now.in_time_zone)
```

Comment:
`freeze_time` uses `Time.now`. I have a question. Are `Time.current`, `Time.zone.now`, and `Time.now.in_time_zone` safe compatibility?

```ruby
def freeze_time(&block)
  travel_to Time.now, &block
end
```

https://github.com/rails/rails/blob/v7.0.3/activesupport/lib/active_support/testing/time_helpers.rb#L234-L236

Reply:
Thank you very much.
I certainly don't think we can say that it is safe compatibility.

https://github.com/rails/rails/blob/3872bc0e54d32e8bf3a6299b0bfe173d94b072fc/activesupport/lib/active_support/core_ext/time/calculations.rb#L38-L41

```ruby
# Returns <tt>Time.zone.now</tt> when <tt>Time.zone</tt> or <tt>config.time_zone</tt> are set, otherwise just returns <tt>Time.now</tt>.
def current
  ::Time.zone ? ::Time.zone.now : ::Time.now
end
```

I'll set `SafeAutoCorrect: false` and leave that as Cop documentation.